src/main/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -220,20 +217,6 @@ protected String downloadSignatureScanner(File scannerExpansionDirectory, HttpUr
         }
     }
 
-    /**
-     * Deletes legacy version file that may be left behind if client had previously installed scan-cli using {@link ZipApiScannerInstaller}
```

Comment:
not sure if this is really a concern, but could we potentially be leaving this file behind before all versions of the CLI move to this new file approach?

Reply:
So with this MR, we no longer have "old version file" vs "new version file".  We continue to use the original file, "blackDuckVersion.txt" as we always have. Meaning: 

Regardless of which API ended up being called, whether through ToolsApiScannerInstaller or ZipApiScannerInstaller, both execution paths in the code will update this version file. Before this MR, I had gone with an approach where I create and maintain a new "scan-cli-version.txt" file for when the new API is being called. This created problems because you had to see "ok am I on the execution path for the new API or the old API? Which file should I read and which file should I make sure to delete?" which complicated the code and Mayank found a bug as well for a use case I hadn't thought of. To fix that bug, after discussing at standup, using just one file would be best